accounts/forms.py

```diff
@@ -327,3 +319,18 @@ def __init__(self, *args, **kwargs):
                 }
         super(DeleteUserForm, self).__init__(*args, **kwargs)
 
+
+class EmailSettingsForm(forms.Form):
+    enabled_stream_emails = forms.BooleanField(
```

Comment:
To what is now `EmailType` right?. That sounds ok, could be called `send_by_default` or something like this.